flask_mongoengine/operation_tracker.py

```diff
@@ -32,7 +33,7 @@
     'update': pymongo.collection.Collection.update,
     'remove': pymongo.collection.Collection.remove,
     'refresh': pymongo.cursor.Cursor._refresh,
-    '_unpack_response': pymongo.helpers._unpack_response,
+    '_unpack_response': pymongo.command_cursor.CommandCursor._unpack_response,
```

Comment:
Notes based on looking at the source code for PyMongo v3.3 (branch for v3.4 doesn't exists on GH) and v3.7:
* [`CommandCursor._unpack_response`](https://github.com/mongodb/mongo-python-driver/blob/v3.3/pymongo/helpers.py#L95-L97) has a different function signature than [`pymongo.helpers._unpack_response`](https://github.com/mongodb/mongo-python-driver/blob/v3.7/pymongo/command_cursor.py#L211). Namely, the former adds `self` as the first argument since this function has now become a method.
* Looking at the logic in that function, the code seems very similar, though I can't with 100% certainty say they're exactly the same.
    * Equivalent parts: [old](https://github.com/mongodb/mongo-python-driver/blob/v3.3/pymongo/helpers.py#L114-L137), [new](https://github.com/mongodb/mongo-python-driver/blob/v3.7/pymongo/message.py#L1375-L1398).
    * The return value of the old function was a [dict](https://github.com/mongodb/mongo-python-driver/blob/v3.3/pymongo/helpers.py#L139-L142).
    * The return value of the new function is 🤷 (whatever [`bson.decode_all(self.documents, codec_options)`](https://github.com/mongodb/mongo-python-driver/blob/v3.7/pymongo/message.py#L1419) returns).